### PR TITLE
Adding vegeta-wrapper dashboard to grafonnet

### PR DESCRIPTION
**Pull request by Manda-supraja26**
## Type of change

- [ ] Refactor
- [X] New feature
- [ ] Bug fix
- [X] Optimization
- [ ] Documentation Update

## Description

Adding vegeta-wrapper dashboard to grafonnet

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
Requesting a link to the previous and current dashboards to make sure they are exactly the same look and feel wise.

**Review by vishnuchalla — CHANGES_REQUESTED** [review]
Requesting a link to the previous and current dashboards to make sure they are exactly the same look and feel wise.

**Comment by Manda-supraja26** [comment]
@vishnuchalla 

old dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/jNFApWhSz/vegeta-results-dashboard?orgId=1

new dashboard:  http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/c6KApZhIk/vegeta-results?orgId=1

**Comment by vishnuchalla** [comment]
> @vishnuchalla
> 
> old dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/jNFApWhSz/vegeta-results-dashboard?orgId=1
> 
> new dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/c6KApZhIk/vegeta-results?orgId=1

Hosting these dashboards on your own cluster is not useful when they have `elasticsearch` as data source. Can you use your dev grafana instance instead to host them for testing and PR review purposes please?

**Comment by Manda-supraja26** [comment]
old dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/dd643298-6931-4e51-8357-780a629fdd68/vegeta-results-dashboard-smanda?orgId=1

new dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/b6882e7b-c4f2-4f02-be3a-a5b771f54454/vegeta-results?orgId=1

File: templates/General/vegeta-wrapper-v2.jsonnet [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  variables.Datasource,
+  variables.uuid,
+  variables.hostname,
+  variables.targets,
+  variables.iteration,
+])
++ g.dashboard.withPanels([
+  panels.timeSeries.legendDisplayModeTable('RPS (rate of sent requests per second)', 'reqps', queries.rps.query(), { x: 0, y: 0, w: 12, h: 9 }),
+  panels.timeSeries.legendDisplayModeTable('Throughput (rate of successful requests per second)', 'reqps', queries.throughput.query(), { x: 12, y: 0, w: 12, h: 9 }),
+  panels.timeSeries.legendDisplayModeTable('Request Latency (observed over given interval)', 'µs', queries.latency.query(), { x: 0, y: 12, w: 12, h: 9 }),
```
Comment (vishnuchalla) [inline templates/General/vegeta-wrapper-v2.jsonnet:29]: Average `p99_latency` is missing here. Can you please get it added as it was there in the previous version?

Comment (Manda-supraja26) [reply templates/General/vegeta-wrapper-v2.jsonnet:29]: I will add it.

**Review by vishnuchalla — APPROVED** [review]
lgtm

**Pull request merged**